### PR TITLE
Implement default enabled color, provide info to callback about defau…

### PR DESCRIPTION
**Pull request by afwn90cj93201nixr2e1re**
Fixes: #888
Fixes: #886

### CODE REVIEW

**Comment by afwn90cj93201nixr2e1re** [comment]
@Arkshine can you merge it?

File: amxmodx/newmenus.cpp [hunk 1]
```diff
@@ -806,14 +809,20 @@ static cell AMX_NATIVE_CALL menu_additem(AMX *amx, cell *params)
...
+	access = params[arg_access];
 
 	menuitem *pItem = pMenu->AddItem(name, cmd, access);
 
-	pItem->handler = params[5];
+	pItem->handler = params[arg_callback];
+
+	if (params[arg_count] / sizeof(cell) >= arg_enabled_color)
+	{
+		char *enabled_color;
```
Comment (Arkshine) [inline amxmodx/newmenus.cpp:823]: Unused.

**Comment by afwn90cj93201nixr2e1re** [comment]
> enabled_color seems quite confusing. If anything, we are talking about the default_active_color.

im pretty agree with you, at all, i also thought about that, i was betwen default_color and active_color, so, you know why i selected active_color? Coz there's condition, that's the only one reason:
https://github.com/alliedmodders/amxmodx/blob/cac06ff4cc2250dab2fcfeb45feaa5fdc944bca9/amxmodx/newmenus.cpp#L498-L502

as you can see it's under condition.
default_active_color - i don't get point, but i can describe why it's can't be 'default', coz by default it was \w, but if someone gonna change it for example with set_itemcolor and blablabla, then it's not default anymore, u agree? So, that's why i chose that.

File: amxmodx/newmenus.cpp (hunk 1 above)
Comment (afwn90cj93201nixr2e1re) [reply amxmodx/newmenus.cpp]: Anything else?

**Comment by afwn90cj93201nixr2e1re** [comment]
I also keep in case that menu item can be '\yA\rblabla', if you're really need renaming just say that again and then  im gonna rename it.

**Comment by afwn90cj93201nixr2e1re** [comment]
Done, anything else? @Arkshine

**Comment by afwn90cj93201nixr2e1re** [comment]
Also If it's possible merge this one, but make them 16 instead 8, just for future using
https://github.com/alliedmodders/amxmodx/pull/656

And then without a twinge of conscience, I will make both default_disabled_color and combine it into one native, now I have not done this just in order not to approach the art (4) limit.

@Arkshine

**Pull request closed without merging**